README.md

```diff
@@ -6,31 +6,31 @@ This Obsidian plugin allows you to import notes from other apps and file formats
 
 You can help! See our [Contribution guidelines](/CONTRIBUTING.md).
 
-- [x] Evernote `.enex`
-- [x] HTML, folder of files
-- [x] Notion, `.zip` of HTML files
-- [x] Bear `.bear2bk`
-- [x] Google Keep
-- [ ] Apple Notes
-- [ ] Microsoft OneNote
-- [ ] Roam Research
-- [ ] Other Markdown flavors
+-   [x] Evernote `.enex`
```

Comment:
I don't think this should be reformatted.

Reply:
Oops